site/content/blogs/release-v127.en.md

```diff
@@ -87,11 +87,74 @@ Continue with the deployment? (y/N)
 
 If the diff looks good to you, enter "y" to deploy. Otherwise, enter "N" to make adjustments as needed!
 
-## Enable multiple listeners and routing rules for Load Balancers
 
-### Add multiple host-based or path-based routing rules to your Application Load Balancers
+## Enable multiple listeners and listener rules for Load Balancers
+You can now expose multiple ports through your Load Balancers.
+### Add multiple host-based or path-based listener rules to your Application Load Balancer
+To expose multiple ports through the Application Load Balancer, we will configure additional listener rules through a new `http` field called [`additional_rules`](../docs/manifest/lb-web-service.en.md#http-additional-rules). 
+It is as easy as configuring your `http` field. Let's learn through an example. 
+
+Say we want to expand the basic manifest such that it opens up port 8081 on the main service container, and 8082 on the sidecar container, in addition to the existing `image.port` 8080.
+```yaml
+# Example 1
+name: 'frontend'
+type: 'Load Balanced Web Service'
+ 
+image:
+  build: './frontend/Dockerfile'
+  port: 8080
+  
+http:
+  path: '/'
+  additional_rules:             # The new field "additional_rules",
+    - target_port: 8081        # Optional. Defaults to the `image.port`.
+      path: 'customerdb'
+    - target_port: 8082
+      target_container: nginx   # Optional. Defaults to the main container. 
+      path: 'admin'
+    - target_port: 80
+      path: 'superAdmin'
```

Comment:
reading the below section, i think it might be clearer if the `path` is the top item in each of the objects